docker/travis_test/Dockerfile

```diff
@@ -1,11 +1,11 @@
 #!BuildTag: openqa_dev
```

Comment:
@ggardet if you keep this file untouched for now I guess we can have the PR merged :)

Reply:
Done